Cargo.toml

```diff
@@ -66,16 +66,16 @@ serial_test = { workspace = true }
 tempfile = { workspace = true }
 
 [workspace.dependencies]
-astarte-device-sdk = "0.9.2"
+astarte-device-sdk = { git = "https://github.com/astarte-platform/astarte-device-sdk-rust", rev = "e8469c09100be2ad5446a7eb3bee359ee8314182"}
```

Comment:
you need to change also the sdk mock with the same info as the astarte-device-sdk